eng/versioning/external_dependencies.txt

```diff
@@ -189,6 +189,7 @@ org.eclipse.aether:aether-transport-wagon;1.1.0
 org.eclipse.jetty:jetty-http;9.4.36.v20210114
 org.eclipse.jetty:jetty-server;9.4.36.v20210114
 org.eclipse.jgit:org.eclipse.jgit;4.5.7.201904151645-r
+org.glassfish:javax.json;1.1.4
```

Comment:
LGTM about the version. Refs: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/scripts/spring_managed_external_dependencies.txt#L482